PRIVACY.md

```diff
@@ -0,0 +1,111 @@
+| [JSTOR](https://www.jstor.org/)                                                                                 | <https://www.ithaka.org/privacypolicy/> |
+| [Library of Congress](https://lccn.loc.gov/)                                                                    | <https://www.loc.gov/legal/> |
+| [National Library of Medicine](https://www.ncbi.nlm.nih.gov/)                                                   | <https://www.nlm.nih.gov/web_policies.html> |
+| [MathSciNet](http://www.ams.org/mathscinet)                                                                     | <https://www.ams.org/about-us/privacy> |
+| [mEDRA](https://medra.org/)                                                                                     | <https://www.medra.org/stdoc/en/Servizio_DOI_Informativa_ENG.pdf> |
+| [Mr. DLib](https://mr-dlib.org/) &ast;                                                                          | <https://support.dataverse.harvard.edu/harvard-dataverse-privacy-policy> |
+| [Openlibrary](https://openlibrary.org)                                                                          | <https://archive.org/about/terms.php> |
+| [ResearchGate](https://www.researchgate.net/)                                                                   | <https://www.researchgate.net/privacy-policy> |
+| [IETF Datatracker](https://datatracker.ietf.org/)                                                               | <https://www.ietf.org/privacy-statement/> |
+| [Semantic Scholar](https://www.semanticscholar.org/), powered by [Allen Institute for AI](https://allenai.org/) | <https://allenai.org/privacy-policy> |
+| [Springer Nature](https://dev.springernature.com/)                                                              | <https://dev.springernature.com/privacypolicy> |
+| [The SAO/NASA Astrophysics Data System](https://ui.adsabs.harvard.edu/)                                         | <https://ui.adsabs.harvard.edu/help/privacy/> |
+| [Unpaywall](https://unpaywall.org/)                                                                             | <https://unpaywall.org/legal/privacy> |
+| [zbMATH Open](https://www.zbmath.org)                                                                           | <https://zbmath.org/privacy-policy/> |
+
+&ast; *Note: The Mr. DLib service is used for the related articles tab in the entry editor and collects also your language, your browser and operating system (by default *disabled*).* 
+
+## JabRef Browser Extension
+
+No personal data (like name, email address, billing address or credit card) is collected.
+By using the extension you will remain completely anonymous.
+
+### Collecting information
+
+When you access a website with your web browser, the Browser Extension is designed to save time by allowing you to quickly import bibliographic information into JabRef.
+Therefor the extension collects and analyzes some information concerning your browser activity.
```

Comment:
The extension does not really collect or store information. 
Suggestion: 
When actively used, the extension has access to the current website and its content. The information is only for the purpose of processing citations.